File: pset6/sentimental/mario.py
from cs50 import get_int

# Recieve user's input 
print("Height should be between 1 and 8.")
height = get_int("Insert Height: ")

# Check that input is bet. 1 & 8
while height <= 0 or height > 8:
    height = get_int("Height: ")

for i in range(height, 0, -1):
    # Print spaces
    x = i - 1
    for j in range(x, 0, -1):
        print(" ", end='')

    # Print hashes
    y = height - x
    for k in range(y, 0, -1):
        print("#", end='')
    
    # new line
    print()

File: pset1/mario.c
// include libraries
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // recieve user's input
    printf("Height should be between 1 and 8.\n");
    int height = get_int("Insert Height: ");

    // make sure that input is bet. 1 & 8
    while (height <= 0 || height > 8)
    {
        height = get_int("Height: ");
    }


    for (int i = height; i > 0; i--)
    {
        // print spaces
        for (int j = i - 1; j > 0; j--)
        {
            printf(" ");
        }

        // print hashes
        for (int k = height - (i - 1); k > 0; k--)
        {
            printf("#");
        }

        // new line
        printf("\n");
    }
}
